src/actions/actionCreators.js

```diff
@@ -447,3 +458,27 @@ export const setNewCategory = (category) => ({
   type: SET_NEW_CATEGORY,
   payload: { category },
 });
+
+export const setClaimVisibility = (claim, value) => ({
+  type: SET_CLAIM_VISIBILITY,
+  payload: { claim, value },
+});
+
+export const updateSelectedClaims = (claim) => ({
+  type: UPDATE_SELECTED_CLAIMS,
+  payload: { claim },
+});
+
+export const clearSelectedClaims = () => ({
+  type: CLEAR_SELECTED_CLAIMS,
+});
+
+export const updateCategoryForClaims = (claims, categoryId) => ({
+  type: UPDATE_CATEGORY_FOR_CLAIMS,
+  payload: { claims, categoryId },
+});
+
+export const moveClaimsToCategory = (targetCategory) => ({
```

Comment:
What claims are you moving? If it's something from the state, make is visible in the name of the action creator.